### PR TITLE
Added Navigation Bar Animation

### PR DESCRIPTION
**Pull request by Rework-Digital**

### CODE REVIEW

File: assets/js/solcast.webflow.footer.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const targetElement = document.querySelector('.footer_subscribe-popup');
+
+if (targetElement) {
+    // Create a new MutationObserver
+    const observer = new MutationObserver((mutations) => {
+        mutations.forEach((mutation) => {
+            if (mutation.type === 'attributes' && mutation.attributeName === 'style') {
+                const displayStyle = window.getComputedStyle(targetElement).display;
+                if (displayStyle === 'flex') {
+                    iFrameResize({log:true});
```
Comment (alex-solcast) [inline assets/js/solcast.webflow.footer.js:12]: The comments here could be more meaningful about WHY we need to resize the footer popup, but mostly I can't see where `iFrameResize` is defined? 

Should this be in the same file or if not, where should other developers look to find it?

Comment (Rework-Digital) [reply assets/js/solcast.webflow.footer.js]: @alex-solcast I've added some additional comments to the js file. I believe these should be 2 separate js files as the iframe.resizer.js script was created by Salesforce.

**Comment by alex-solcast** [comment]
Thanks Liam,

It's telling me there are merge conflicts that you'll need to resolve
before I can merge it:

[image: image.png]

**Comment by Rework-Digital** [comment]
@alex-solcast Thanks, Alex. I have fixed up the conflict.

**Pull request merged**